syntax = "proto3";

service FileService {
  rpc DownloadFile (FileRequest) returns (FileResponse);
}

message FileRequest {
  string file_name = 1;
}

message FileResponse {
  string file_content = 1;
}
